pkg/controllers/provisioning/v1alpha1/reallocation/annotate.go

```diff
@@ -31,11 +31,11 @@ type Annotator struct {
 }
 
 // MarkUnderutilized takes in a list of underutilized nodes and adds TTL to them
-func (a *Annotator) MarkUnderutilized(ctx context.Context, nodes []*v1.Node) error {
+func (a *Annotator) MarkUnderutilized(ctx context.Context, nodes []*v1.Node, ttlSeconds int32) error {
```

Comment:
May be we should pass in the provisioner object here, like we do in other cases. We might need some more config later in the annotator, but, happy to punt to next time when we add more config here.

Reply:
I was going on the side of not wanting to pass in large objects into the function if I didn't have to. Passing in the TTL here is sufficient.